tests/test_pycdf_istp.py

```diff
@@ -523,6 +524,14 @@ def testValidScale(self):
              'SCALEMIN type CDF_INT2 does not match variable type CDF_BYTE.'
              ],
              errs)
+        tt2000.attrs['SCALEMIN'] = 200.42e45 #tt2000 testing
+        errs = spacepy.pycdf.istp.VariableChecks.validscale(tt2000)
+        errs.sort()
+        self.assertEqual(
+            ['SCALEMIN type CDF_DOUBLE does not match variable type CDF_TIME_TT2000.',
+             'SCALEMIN type CDF_DOUBLE not comparable to variable type CDF_TIME_TT2000.',
```

Comment:
Aha, this is weird! I hadn't realized that the code would actually register a validation failure at this point, but keep going...since the exception then hid that. Nice, this is definitely the behavior we want.